extensions/amp-analytics/amp-analytics.md

```diff
@@ -398,6 +398,32 @@ As an example, the following configuration can be used to sample 50% of the requ
 },
 ```
 
+#### Embed render start trigger (`"on": "render-start"`)
+Use this configuration to fire a request when this document's or a specified embed's receive render-start signal. This
```

Comment:
There are some grammatical issues here.

I think what would really help is to start out by describing what an embed (give ad example) and a document is.